utils/learn-lm.json

```diff
@@ -561,12 +561,12 @@
             ]
         },
         {
-            "dirName": "Featured use cases",
+            "dirName": "Featured scenarios",
             "level": 1,
             "position": 4,
             "isDir": true,
-            "url": "swan-lake/featured-use-cases",
-            "id": "featured-use-cases",
+            "url": "swan-lake/featured-scenarios",
```

Comment:
We cannot change the `url` & `id` without renaming the associated directory

Reply:
Will be fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/6828.